e2e-tests/functions

```diff
@@ -183,6 +187,14 @@ create_infra() {
     local ns="$1"
 
     if [ -n "$OPERATOR_NS" ]; then
+        kubectl get pxc --all-namespaces -o wide \
+            | grep -v 'NAMESPACE' \
+            | xargs -L 1 sh -xc 'kubectl patch pxc -n $0 $1 --type=merge -p "{\"metadata\":{\"finalizers\":[]}}"' \
```

Comment:
null also fits here

Reply:
it is possible to do in different ways, `[]` works well.
Is any reason why we should use `null` here?